tests/appsec/iast/taint_sinks/test_sql_injection.py

```diff
@@ -64,9 +65,8 @@ def test_sql_injection(fixture_path, fixture_module, iast_span_defaults):
     assert vulnerability.hash == hash_value
 
 
-@pytest.mark.parametrize("num_vuln_expected", [1, 0, 0])
 @pytest.mark.parametrize("fixture_path,fixture_module", DDBBS)
-def test_sql_injection_deduplication(fixture_path, fixture_module, num_vuln_expected, iast_span_deduplication_enabled):
+def test_sql_injection_deduplication(fixture_path, fixture_module, iast_span_deduplication_enabled):
```

Comment:
Is this testing deduplication now if we only run it once?

Reply:
Yes: https://github.com/DataDog/dd-trace-py/pull/9071/files#diff-a1c329964168c2e950459bed80e06b63c03be9d9886c9ff3570c912f97e70023R79